global/config.cpp

```diff
@@ -216,6 +217,9 @@ void Config::load(QString path) {
         } else if (stream.name() == "WaveformResolution") {
           stream.readNext();
           waveform_resolution = stream.text().toInt();
+        } else if (stream.name() == "DefaultImageDuration") {
+          stream.readNext();
+          default_image_duration = QTime::fromString(stream.text().toString(), olive::kDefaultTimeFormat);

```

Comment:
Look like there are no guards against bad config values (not just here, but in general), e.g. a negative image default duration

Reply:
Well, that's what I noticed as well. I think we need a better config handling in general.